h/views/accounts.py

```diff
@@ -487,15 +487,9 @@ class AccountController(object):
     def __init__(self, request):
         self.request = request
 
-        save_email_label = _('Change email address')
-        save_password_label = _('Change password')
-
-        if request.feature('activity_pages'):
-            email_schema = schemas.EmailChangeSchema().bind(request=request)
-            save_email_label = _('Save')
-            save_password_label = _('Save')
-        else:
-            email_schema = schemas.LegacyEmailChangeSchema().bind(request=request)
+        email_schema = schemas.EmailChangeSchema().bind(request=request)
+        save_email_label = _('Save')
+        save_password_label = _('Save')
```

Comment:
We probably don't need these to variables anymore and can just pass `_('Save')` to the buttons argument in the two `create_form` calls further down.